wagn/lib/wagn/commands.rb

```diff
@@ -36,19 +36,27 @@ def find_spec_file filename, base_dir
   end
 end
 
+LOAD_TASKS= {
+  'reseed' => 'create',
+  'seed' => 'recreate',
+  'load' => 'seed'
+}
 
 if supported_rails_command? ARGV.first
   if ARGV.delete('--rescue')
     ENV["PRY_RESCUE_RAILS"]="1"
   end
-  require 'wagn'
+  command = ARGV.first
+  command = ALIAS[command] || command
+  require 'generators/card' if command == 'generate' # without this, the card generators don't list with: wagn g --help
   require 'rails/commands'
 else
   command = ARGV.shift
   command = ALIAS[command] || command
 
   case command
-  when 'seed'
+  when 'seed', 'reseed', 'load'
+    task = LOAD_TASKS[command]
```

Comment:
Yeah, kind of critical to know what tasks run when.  And I agree once we settle the API the mapping shouldn't be needed.